### PR TITLE
V6 ontap callback

### PR DESCRIPTION
**Pull request by upatel32**
Add support for an `onTap` prop along with an appropriate example.

### CODE REVIEW

File: migration.md [hunk 1]
```diff
@@ -1,8 +1,10 @@
-React Swipeable v6 changes and migration
-=========================
+# React Swipeable v6 changes and migration
 
 ## Major Changes
+
 - removal of `<Swipeable>` component
   - _TODO_ add example for creating one and migrating
```
Comment (hartzis) [inline migration.md:7]: 💭 add new section for `New Features` and add this to that list, and remove from migrate section

File: src/index.ts [hunk 2]
```diff
@@ -227,16 +230,18 @@ function getHandlers(
 
   const onEnd = (event: HandledEvents) => {
     set((state, props) => {
-      let eventData: EventData | undefined;
-      if (state.swiping) {
-        eventData = { ...state.eventData, event } as EventData;
+      //let eventData: EventData | undefined;
+      const eventData = { ...state.eventData, event } as EventData;
 
+      if (state.swiping) {
```
Comment (hartzis) [inline src/index.ts:236]: ❌ put back.
- we do not want to save the `eventData` if it was not a swipe

Comment (upatel32) [reply src/index.ts]: put things back to how they were and left the else statement

File: migration.md (hunk 1 above)
Comment (upatel32) [reply migration.md:11]: added to `New Features` section

**Review by hartzis — APPROVED** [review]

**Pull request merged**